pkg/loadbalancer/experimental/bpf_reconciler.go

```diff
@@ -430,6 +438,7 @@ func (ops *BPFOps) Prune(_ context.Context, _ statedb.ReadTxn, _ iter.Seq2[*Fron
 		ops.pruneBackendMaps(),
 		ops.pruneRevNat(),
 		ops.pruneSourceRanges(),
+		// TODO: Should there be any pruning for Maglev? If so, probably BPFOps should have additional fields, on which pruning would be based.
```

Comment:
Yeah should add pruning for maglev as well. Should be easy enough: dump the outer map and delete each outer value that points to a non-existing frontend ID. Almost identical to `pruneRevNat`.

What do you mean by additional fields? I don't see why this needs any extra state.

Reply:
> What do you mean by additional fields? I don't see why this needs any extra state.

Other pruning methods rely on `ops.backendReferences` or `ops.backendStates`, so I was envisioning that something similar may be needed, but your previous paragraphs clarified that it's not. Thanks!